File: AHome/27.01/phpFullStack/API/PersonDAO/Model/Person.php
<?php
	class Person {
       
       public function __construct($id, $firstName, $lastName, $age){
            $this->id = $id;
            $this->firstName = $firstName;
            $this->lastName = $lastName;
            $this->age = $age;
       }
       
	   public $id;
       public $firstName;
       public $lastName;
       public $age;
    }
?>

File: AHome/03.02/csClient/PersonForm/PersonForm/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PersonForm
{
    public class Person
    {
        public int id { get; set; }
        public string firstName { get; set; }
        public string lastName { get;  set; }
        public int age { get; set; }
        
        public Person(){}

        public Person(int id, string firstName, string lastName, int age)
        {
            this.id = id;
            this.firstName = firstName;
            this.lastName = lastName;
            this.age = age;
        }

        override public string ToString()
        {
            string res = "";

            res = "[" + id + "] [" + firstName + "] [" + lastName + "] [" + age + "]";

            return res;
        }
    }
}
